letsencrypt_nginx/configurator.py

```diff
@@ -274,17 +291,22 @@ def _make_server_ssl(self, vhost):
         :type vhost: :class:`~letsencrypt_nginx.obj.VirtualHost`
 
         """
-        ssl_block = [['listen', '443 ssl'],
-                     ['ssl_certificate',
-                      '/etc/ssl/certs/ssl-cert-snakeoil.pem'],
-                     ['ssl_certificate_key',
-                      '/etc/ssl/private/ssl-cert-snakeoil.key'],
+        snakeoil_cert, snakeoil_key = self._get_snakeoil_paths()
+        ssl_block = [['listen', '{0} ssl'.format(self.config.dvsni_port)],
```

Comment:
That's a good point. Using `config.dvsni_port` everywhere was the fastest way to get integration testing working. I think we should also have `--nginx-ssl-vhost-port` (default `443`), but hardcoding must be avoided. Follow-up PR?